HW02/Lizaveta_Muromskaya/spec/my_array_methods_spec.rb

```diff
@@ -0,0 +1,100 @@
+# frozen_string_literal: true
+
+require 'rspec'
+require_relative 'my_array_methods'
+using MyArrayMethods
+
+RSpec.describe 'MyArrayMethods' do
+  empty_array = []
+  array = [1, 2, 3, 4, 5]
+
+  describe '.my_each' do
```

Comment:
https://www.betterspecs.org/
<img width="1143" alt="Screenshot 2021-05-22 at 13 38 43" src="https://user-images.githubusercontent.com/12089948/119223602-0948e400-bb03-11eb-8bfe-c01a4ef5a396.png">

```suggestion
  describe '#my_each' do
```